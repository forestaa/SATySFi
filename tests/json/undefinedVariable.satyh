let x = f 3
