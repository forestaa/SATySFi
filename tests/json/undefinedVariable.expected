{
  "file": "undefinedvariable.satyh",
  "range": {
    "start": { "line": 1, "character": 8 },
    "end": { "line": 1, "character": 9 }
  },
  "message": "! [Type Error] undefined variable 'f'."
}
